components/touchscreen/lilygo_t5_47.rst

```diff
@@ -24,6 +24,7 @@ Configuration variables:
 - **id** (*Optional*, :ref:`config-id`): Manually set the ID of this touchscreen.
 - **interrupt_pin** (*Optional*, :ref:`Pin Schema <config-pin_schema>`): The touch detection pin.
   Must be ``GPIO13``.
+- **touch_timeout** (*Optional*, :ref:`config-time`): The time to automatically check if touch was released. Defaults to ``250ms``.
```

Comment:
Since the touch_timeout is a generic touchscreen configuration option, and there is already the "All other options from config-touchscreen" reference below; I'm not sure it is necessary to explicitly add that one here.
```suggestion
```

Reply:
I agree with @guillempages.

Reply:
done